third_party/curl/CMakeLists.txt

```diff
@@ -9,8 +9,9 @@ list(APPEND CMAKE_ARGS
     "-DCMAKE_TOOLCHAIN_FILE:PATH=${CMAKE_TOOLCHAIN_FILE}"
     "-DCMAKE_POSITION_INDEPENDENT_CODE=ON"
     "-DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE}"
-    "-DCURL_USE_OPENSSL=OFF"
+    "-DCURL_USE_OPENSSL=ON"
     "-DCURL_USE_LIBSSH2=OFF"
+    "-DBUILD_CURL_EXE=OFF"
```

Comment:
I don't see a need to disable that only for Android and iOS, and disabling it pushes the need for dlopen to the build step, which is convenient.